packages/multi-select-combo-box/theme/lumo/vaadin-multi-select-combo-box-chip-styles.js

```diff
@@ -13,27 +13,65 @@ import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themab
 
 const chip = css`
   :host {
-    display: inline-flex;
-    align-items: center;
-    align-self: center;
     font-family: var(--lumo-font-family);
     font-size: var(--lumo-font-size-xxs);
     line-height: 1;
-    padding: 0.3125em 0 0.3125em calc(0.5em + var(--lumo-border-radius-s) / 4);
+    padding: 0.3125em calc(0.5em + var(--lumo-border-radius-s) / 4);
     border-radius: var(--lumo-border-radius-s);
     border-radius: var(--lumo-border-radius);
```

Comment:
Is this on purpose, for some kind of backwards compatibility? I would remove it.
```suggestion
```

Reply:
This was probably a copy-paste. Removed.